tests/ci_visibility/test_ci_visibility.py

```diff
@@ -220,82 +258,6 @@ def test_repository_name_not_extracted_warning():
 DUMMY_RESPONSE = Response(status=200, body='{"data": [{"type": "commit", "id": "%s", "attributes": {}}]}' % TEST_SHA_1)
 
 
-@mock.patch("ddtrace.internal.ci_visibility.recorder._do_request")
```

Comment:
Why do we delete these tests?

Reply:
IMO, they were subsumed by existing tests:

* The `CIVisibility` service's interactions with the `CIVisibilityGitClient` client are covered by [TestCheckEnabledFeatures](https://github.com/DataDog/dd-trace-py/blob/main/tests/ci_visibility/test_ci_visibility.py#L583)
* The `CIVisibilityGitClient`'s upload behavior is tested in [TestUploadGitMetadata](https://github.com/DataDog/dd-trace-py/blob/main/tests/ci_visibility/test_ci_visibility.py#L885)

The use of the `CIVisibilityGitClient` changed in #7894 , where we optionally wait for the client to finish, and test for timeouts, which covers the `.disable()` test here.